### PR TITLE
Add history-rm command

### PR DESCRIPTION
**Pull request by fapfaff**
Implements #308

### CODE REVIEW

File: dunstctl [hunk 1]
```diff
@@ -26,6 +26,8 @@ show_help() {
 	  history-pop [ID]                  Pop the latest notification from
 	                                    history or optionally the
 	                                    notification with given ID.
+	  history-rm [ID]					Remove the notification from
```
Comment (bebehei) [inline dunstctl:29]: Please do not use tabs for separation. This must be spaces.

File: src/dbus.c [hunk 2]
```diff
@@ -397,6 +402,23 @@ static void dbus_cb_dunst_NotificationPopHistory(GDBusConnection *connection,
         g_dbus_connection_flush(connection, NULL, NULL, NULL);
 }
 
+static void dbus_cb_dunst_NotificationRemoveFromHistory(GDBusConnection *connection,
+                                           const gchar *sender,
```
Comment (bebehei) [inline src/dbus.c:406]: Please align the arguments.

**Comment by fwsmit** [comment]
Looks good. There's only a warning from the compiler when compiling the test. Could you fix that?

**Comment by fapfaff** [comment]
> Looks good. There's only a warning from the compiler when compiling the test. Could you fix that?

Should be fixed now

**Comment by fwsmit** [comment]
You have some memory leaks, probably in the tests. You can check this locally with `make test-valgrind`

**Comment by fapfaff** [comment]
It's fixed

**Comment by fwsmit** [comment]
Merged, thank you!

**Pull request merged**